zone/spells.cpp

```diff
@@ -3195,6 +3195,11 @@ bool Mob::CheckSpellLevelRestriction(Mob *caster, uint16 spell_id)
 	bool check_for_restrictions = false;
 	bool can_cast = true;
 
+	if (spells[spell_id].target_type == ST_Self) {
```

Comment:
Maybe put above variable definitions just so they’re not needlessly defined? Looks good otherwise.

Reply:
Done.